bucket/helm.json

```diff
@@ -1,26 +1,23 @@
 {
-    "homepage": "https://helm.sh/",
-    "description": "The package manager for Kubernetes.",
+    "homepage": "https://helm.sh",
+    "description": "The package manager for Kubernetes",
     "license": "Apache-2.0",
-    "version": "2.16.1",
+    "version": "3.0.0",
     "architecture": {
         "64bit": {
-            "url": "https://storage.googleapis.com/kubernetes-helm/helm-v2.16.1-windows-amd64.tar.gz",
-            "hash": "7ab28696e2f5a5b56001aacbe7156aae83c8fbd05946164efa3a2707aac33255"
+            "url": "https://storage.googleapis.com/kubernetes-helm/helm-v3.0.0-windows-amd64.zip",
```

Comment:
Why googleapis, when in autoupdate there is some `get.helm.sh`

Reply:
Sorry, a mistake..